adafruit_max31856.py

```diff
@@ -358,11 +371,11 @@ def oneshot_pending(self) -> bool:
         )
         return bool(oneshot_flag)
 
-    def _wait_for_oneshot(self):
+    def _wait_for_oneshot(self) -> None:
         while self.oneshot_pending:
             sleep(0.01)
 
-    def _read_register(self, address, length):
+    def _read_register(self, address: int, length: int) -> int:
```

Comment:
Since it's actually returning a slice of a `bytearray`, the return here should be `bytearray`.